python/ql/src/Statements/ReturnOrYieldOutsideOfFunction.qhelp

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>In Python, <code>return</code> and <code>yield</code> statements as well as <code>yield from</code>
```

Comment:
I think this should be either 
"as well as the `yield from` expression" 
or 
"as well as `yield from` expressions" 
in order for this sentence to scan correctly.

Reply:
Done